ProjectWithUMLDiagram/src/com/beyond/person/core/service/getMethods/GetAllPerson.java

```diff
@@ -0,0 +1,84 @@
+package com.beyond.person.core.service.getMethods;
+
+import com.beyond.person.core.enumaration.Designation;
+import com.beyond.person.core.model.BasePerson;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class GetAllPerson implements GetMethodsInterface{
+    private static Map<Integer, BasePerson> personsMap;
+    private Map<Integer, BasePerson> personsMapForRemove = new HashMap<>();
+    private  List<BasePerson> listForRemovedElement = new ArrayList<>();
+
+    public GetAllPerson(Map<Integer, BasePerson> personsMap){
+        this.personsMap = personsMap;
+        personsMapForRemove = personsMap.entrySet()
```

Comment:
@AraChilingaryan why you change personMap to Set and after that again to Map ?

Reply:
done